v5/tutorials/general/coding-faq.rst

```diff
@@ -2,6 +2,15 @@
 Coding FAQ
 ==========
 
+    * ``"main.h" file not found``:
+        The linter being separate from the compiler produces this error when

```

Comment:
Not sure how important it is to mention that the linter is separate from the compiler here

```suggestion
This error can occur when the compilation cache
```